sdk/iot/core/tests/cmocka/az_iot_telemetry_tests.c

```diff
@@ -87,7 +83,8 @@ void test_az_iot_hub_client_telemetry_publish_topic_get_NULL_client_fails(void**
 
   uint8_t mqtt_topic_buf[TEST_MQTT_SPAN_BUFFER_SIZE];
 
-  az_span mqtt_topic = az_span_init(mqtt_topic_buf, 0, sizeof(mqtt_topic_buf) / sizeof(mqtt_topic_buf[0]));
+  az_span mqtt_topic
+      = az_span_init(mqtt_topic_buf, 0, sizeof(mqtt_topic_buf) / sizeof(mqtt_topic_buf[0]));
```

Comment:
It would be good to have a [_countof(array)](https://stackoverflow.com/questions/4415524/common-array-length-macro-for-c). 
@barcharcraz, @antkmsft /cc @JeffreyRichter should we define our own or add to core internal? Does something like that exist already and we've missed it?

Reply:
Just as a note I have a list of things to bring up with @JeffreyRichter and friends. Feedback from azrtos team and others. Likely will be for Friday.